### PR TITLE
Merged and rearanged BOTW workaounds 

### PR DESCRIPTION
**Pull request by Zalnor**
Merged the BOTW workaounds into 3 packs, one for each GPU vendor. They all have needed shaders and I added a comment to each one for what pack they were from. Each pack also contains the universal workaounds like clouds, Kakariko torch, and Lwzx crash fix.
I also moved and updated version number for the normal Lwzx crash fix pack to a Vulkan catagory as its currently the only "needed" workaround pack we have for vulkan

### CODE REVIEW

File: Workarounds/BreathOfTheWild_AMD_Workaounds_OGL/rules.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Workarounds for AMD GPUs"
+path = "The Legend of Zelda: Breath of the Wild/Workarounds OpenGL/AMD GPU"
+description = Contains these workaounds for BOTW and AMD GPUs, AMD Shader Crash, AMD Shadows, Kakariko Torch Shadows, Streched Clouds, LWZXNullCheck.
```
Comment (slashiee) [inline Workarounds/BreathOfTheWild_AMD_Workaounds_OGL/rules.txt]:
```
path = "The Legend of Zelda: Breath of the Wild/Workarounds OpenGL/AMD GPU for OpenGL"
```
`Contains these workarounds for BOTW and AMD GPUs: AMD Shader Crash,` etc...

File: Workarounds/BreathOfTheWild_Nvidia_Workaounds_OGL/rules.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Workarounds for Nvidia GPUs"
+path = "The Legend of Zelda: Breath of the Wild/Workarounds OpenGL/Nvidia GPU"
+description = Contains these workaounds for BOTW and Nvidia GPUs,Nvidia Smoke, Kakariko Torch Shadows, Streched Clouds, LWZXNullCheck.
```
Comment (slashiee) [inline Workarounds/BreathOfTheWild_Nvidia_Workaounds_OGL/rules.txt]:
```
name = "Workarounds for NVIDIA GPUs"
path = "The Legend of Zelda: Breath of the Wild/Workarounds/NVIDIA GPU for OpenGL"
```
`Contains these workarounds for BOTW and NVIDIA GPUs: NVIDIA Smoke, Kakariko Torch Shadows,` etc...

File: Workarounds/BreathOfTheWild_LwzxNullCheck_VK/rules.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+path = "The Legend of Zelda: Breath of the Wild/Workarounds Vulkan/LWZX Crash"
```
Comment (slashiee) [inline Workarounds/BreathOfTheWild_LwzxNullCheck_VK/rules.txt]: Is this pack really necessary for Vulkan? I've been playing the game in Vulkan without this pack and have encountered zero crashes.

If you want to keep it though, fix the metadata.
```
name = "LWZX Crash"
path = "The Legend of Zelda: Breath of the Wild/Workarounds/LWZX Crash for Vulkan"
```

Also, don't include this crash workaround in all the packs.

Comment (Zalnor) [reply Workarounds/BreathOfTheWild_LwzxNullCheck_VK/rules.txt]: it was needed for if users dont have Full sync at GX2 drawdone on the game can crash. we cant really test it in vulkan as we need it on for the games to run, but in opengl its needed any time they have it off.

**Pull request closed without merging**